chromium_src/components/permissions/request_type.h

```diff
@@ -6,9 +6,10 @@
 #ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_PERMISSIONS_REQUEST_TYPE_H_
 #define BRAVE_CHROMIUM_SRC_COMPONENTS_PERMISSIONS_REQUEST_TYPE_H_
 
-#define kStorageAccess                                     \
-  kStorageAccess, kWidevine, kBraveEthereum, kBraveSolana, \
-      kBraveGoogleSignInPermission, kBraveLocalhostAccessPermission
+#define kStorageAccess                                               \
+  kStorageAccess, kWidevine, kBraveEthereum, kBraveSolana,           \
+      kBraveGoogleSignInPermission, kBraveLocalhostAccessPermission, \
+      kBraveDefault
```

Comment:
this doesn't seem right to me cc @goodov

Reply:
I'm not sure why we're getting the default content setting here in permissions, but it's also not a  brave specific permission so we shouldn't require a new request type here. I think we must be handling something incorrectly. Can this be replicated on desktop under the same conditions?

Reply:
the crash is just uma stuff, but it seems like there is some other underlying issue if we're getting DEFAULT as the dismissal type. I feel like this change might just be covering up a more serious problem

Reply:
we are getting DEFAULT as we need to register Widevine Content Settings Type here otherwise https://github.com/brave/brave-core/blob/master/chromium_src/components/content_settings/core/common/content_settings_types.mojom and here https://github.com/brave/brave-core/blob/master/chromium_src/components/content_settings/core/browser/content_settings_registry.cc, so the conversion from Content Settings Type to Registry Type for Widevine works. I don't think we want that as you mentioned it's only for UMA which we don't need.

Reply:
I open for suggestions on how we can resolve different and fast without delaying cr126 update.

Reply:
chromium didn't have that UMA check before cr126 that's why it worked without any issues.

Reply:
I see what's going on here now, we don't have a content setting for widevine so it's mapping to `ContentSettingsType::DEFAULT`. As discussed for now let's just map `DEFAULT` <-> `kWidevine` and we'll follow-up to figure out if we want to keep that, add a widevine content setting or do something else

Reply:
I created a follow to think about that https://github.com/brave/brave-browser/issues/38304